files/auto/autopause-fcns.sh

```diff
@@ -17,13 +17,13 @@ rcon_client_exists() {
 }
 
 mc_server_listening() {
-  mc-monitor status --host localhost --port "$SERVER_PORT" --timeout 10s >& /dev/null
+  mc-monitor status "${MC_HEALTH_EXTRA_ARGS[@]}" --host localhost --port "$SERVER_PORT" --timeout 10s >& /dev/null
```

Comment:
The idea of this change is great, but I don't the think the syntax or usage is correct. Have you tested this?

`MC_HELATH_EXTRA_ARGS` is being used as an array type since normally it is declared in `/data/.mc-health.env` here

https://github.com/itzg/docker-minecraft-server/blob/7419a668431e5e13832122d7295d66f10883ae86/scripts/start-finalExec#L226-L229

however, the file `.mc-health.env` file doesn't seem to be getting source'd in the autopause scripts. So that needs to be added, such as [this](https://github.com/itzg/docker-minecraft-server/blob/dca29dcbb5d357811e5114b23f16e59126069a54/bin/mc-health#L5-L7)

Setting the env var `MC_HEALTH_EXTRA_ARGS` is not correct since env variables only have string values, not array values. So the array expansion `${MC_HEALTH_EXTRA_ARGS[@]}` will not be correct. 

I would suggest introducing a new env var named something like `USES_PROXY_PROTOCOL` that is "true" or "false". In the `start-finalExec` location above add a conditional on `isTrue "$USES_PROXY_PROTOCOL"` and add `--use-proxy` to `MC_HEALTH_EXTRA_ARGS` there.

Reply:
Yes, before making this pull request I've tested it with the --use-proxy option but that was about it.

Reply:
If you wish I can make the changes you mentioned, but it will probably be in about 1.5 weeks time from now as I'm working on other projects at the moment.

Reply:
Yes, making those changes would be good. mc-health can also make use of that option then. I don't mind waiting for you to make that change.

BTW, `MC_HEALTH_EXTRA_ARGS` was purposely undocumented since it was only meant to communicate internally the start-time discovery for legacy ping protocol. So, it'll be great to document the new variable since it actually should be visible to users.